dragonphy/jtag.py

```diff
@@ -11,10 +11,15 @@ def __init__(self,  filename=None, **system_values):
         id_code = self.calc_id_code()
         print(f'JTAG ID Code: 0x{id_code:08x}')
 
+        # get a list of markdown files and sort by name
```

Comment:
I wanted to make the ordering of registers more predictable, so I am sorted the Markdown files by name.